zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/BrokerUserCreateRequest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.impl.broker.request;
+
+import io.camunda.zeebe.broker.client.api.dto.BrokerExecuteCommand;
+import io.camunda.zeebe.protocol.Protocol;
+import io.camunda.zeebe.protocol.impl.record.value.user.UserRecord;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.UserIntent;
+import org.agrona.DirectBuffer;
+
+public class BrokerUserCreateRequest extends BrokerExecuteCommand<UserRecord> {
+
+  private final UserRecord requestDto = new UserRecord();
+
+  public BrokerUserCreateRequest() {
+    super(ValueType.USER, UserIntent.CREATE);
+    setPartitionId(Protocol.DEPLOYMENT_PARTITION);
```

Comment:
❓ Is there a specific reason we always send the user create command to the deployment partition?

Reply:
This is something that roman and I discussed but I am unfortunately not able to find the context now. As he is currently on FTO getting that answer soon won't be possible.

My questions would be: is there anything wrong with doing this? Does t seem like a strange choice? Would you recommend a different approach?

Reply:
Wrong is a big word. It would work fine this way. The reason I ask is because it means every user creation will be send to the same partition, instead of it being distributed amongst different partitions. This means that single partition will receive more load than others which could impact performance. I'm not too worried about this as I don't expect there'd be an insane amount of these command being sent.
Another thing is that partition 1 could be unavailable, for example due to backpressure. This means all create commands will get rejected, as opposed to another partition being able to handle them.

In general we should always try to distribute the load evenly across partitions. So even if the impact here wouldn't be that large it's still the nicer thing to do.

Reply:
I have no reason to not go with your suggestion here so lets use the preferred approach of distributing the command, if this needs changing its something that we can address later on :)